### PR TITLE
redirect to logfiles in start script.

### PR DESCRIPTION
**Pull request by endrawes0**

### CODE REVIEW

File: start [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-node chesster.js $1
+node chesster.js $1 >> stdout_log 2>> stderr_log
```
Comment (aweidner) [inline start]: I believe these files should be added to the .gitignore.

**Comment by aweidner** [comment]
One thing to consider is that chesster shouldn't currently write anything to stderr.  I couldn't find any uses of `console.error` in the codebase, everything is currently printing to stdout through `console.log`.     There's also a file called `errors_log` which looks like it contains stacktraces.  What should go into the stderr_log file and what should go into errors_log, do we need both?

File: start (hunk 1 above)
Comment (lakinwecker) [inline start]: I think I'd prefer to have a single log for all of it, something like

```2&>1 >> output.log```

I don't want to have to remember which log to look at for which information.

**Comment by lakinwecker** [comment]
Long term we may want to switch to a proper logging library, but I think a simpler fix, like this, should be good for now

Example Logging library:  https://github.com/winstonjs/winston

File: start (hunk 1 above)
Comment (endrawes0) [inline start]: Hm, I'm ok with that - any logging system your are familiar with? This goes above and beyond the requirements, but could be good to tag the beginning of each line with a timestamp and file?

`2016-05-14T16:13:58+00:00 - stdout: something something something `

**Comment by endrawes0** [comment]
Just saw the reference to Winston

File: start (hunk 1 above)
Comment (endrawes0) [inline start]: Agreed about the ignore

**Comment by endrawes0** [comment]
In my future commits, I will utilize console.error and replace the error log.

for now, just redirecting

**Comment by endrawes0** [comment]
updated and ready for merging if there are no other issues.

**Comment by lakinwecker** [comment]
:+1:

**Comment by endrawes0** [comment]
ok - I lied. I needed to add to .gitignore.

**Pull request merged**